eng/Versions.props

```diff
@@ -70,7 +71,38 @@
     <MicrosoftNETCoreILAsmVersion>5.0.0-preview.4.20202.18</MicrosoftNETCoreILAsmVersion>
     <MicrosoftNETSdkILVersion>5.0.0-preview.4.20202.18</MicrosoftNETSdkILVersion>
     <!-- Libraries dependencies -->
+    <SystemBuffersVersion>4.5.1</SystemBuffersVersion>
```

Comment:
How are we ensuring that these versions are kept up-to-date? Are we using darc? If so, what if some packages are only serviced out of a different branch, for example release/2.1? How will we get an update?

Reply:
For 3.1 we will use BAR and Darc. That isn't possible for 2.1 unfortunately so we need to update these versions manually. Nothing changes here, these versions were already hardcoded in the depproj restore: https://github.com/dotnet/runtime/blob/e0e752ac9ee67f3dc7fa127d6f7495132b4bcee7/src/libraries/restore/binplacePackages/binplacePackages.depproj#L31-L42. I'm just moving them into Versions.props.

FWIW we could consider using something like dependabot.